### PR TITLE
Fzampori csoar changelog

### PR DESCRIPTION
**Pull request by fzampori**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: blog-csoar/2023-11-15-application-update.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: November 15, 2023 - Application Update
+hide_table_of_contents: false
+keywords:
+  - sumo logic
+  - cloud soar
+  - automation service
+ 
+tags: [cloud soar release notes,automation service , application update]
```
Comment (kimsauce) [inline blog-csoar/2023-11-15-application-update.md:9]:
```suggestion
tags: [cloud soar release notes, automation service, application update]
```

File: blog-csoar/2023-11-15-application-update.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+### Changes and Enhancements 
+* Added documentation for [Cloud SOAR Audit Logging](/docs/cloud-soar/audit-event-index/)
+
+### Bug fixes 
+* Actions: Fixed run action causing page reload when response data is too large.
+* Playbooks: Removed Resource from inputs when selecting an Internal integration in add or edit node.
+* Playbooks actions: Fixed boolean values processed as null.
+#### Cloud SOAR
+* Fixed ApiV3 change incident owner when using incorrect owner ID or with a Group ID.
```
Comment (kimsauce) [inline blog-csoar/2023-11-15-application-update.md:23]:
```suggestion
* Fixed API v3 change incident owner when using incorrect owner ID or with a Group ID.
```

File: blog-csoar/2023-11-15-application-update.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: November 15, 2023 - Application Update
```
Comment (kimsauce) [inline blog-csoar/2023-11-15-application-update.md:2]: Should this be 11/16?

**Review by kimsauce — CHANGES_REQUESTED** [review]
Question and suggestions

File: blog-csoar/2023-11-15-application-update.md (hunk 3 above)
Comment (fzampori) [reply blog-csoar/2023-11-15-application-update.md:2]: yes, I'm updating filename too

**Review by jpipkin1 — APPROVED** [review]
It looks good, @fzampori. Approved.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**